user_manual/files/webgui/custom_groups.rst

```diff
@@ -9,6 +9,10 @@ This wasn't the most efficient way to work.
 To address that, as of ownCloud 10.0, you can now create your own groups on-the-fly, through a feature called "Custom Groups". 
 Here's how to use it. 
 
+.. note::
+   	- creating or renaming a custom group using an existing display name will show an error.
```

Comment:
@pmaier1 @mmattel @voroyam - are you ok with the revised wording?

Reply:
just read the wording - what was rephrased?